testing/api/test_utils_shell.py

```diff
@@ -23,7 +23,7 @@ def test_sct_argument_parser(capsys):
 
     # Check that new defaults are set properly
     parser3 = shell.SCTArgumentParser()
-    assert parser3.prog == "test_utils_shell"
+    # assert parser3.prog == "test_utils_shell"  this may actually be `pytest`, `sct_testing`, etc. -- hard to test
```

Comment:
Some parts of this test are clearly written to match the code, rather than being independent requirements/specs. We could probably get rid of those low-value parts.